examples/python/sentiment/main.py

```diff
@@ -1,20 +1,19 @@
-from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
 from fastapi import FastAPI
 from pydantic import BaseModel
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
 
 class PolarityScores(BaseModel):
     pos: float
     compound: float
     neu: float
     neg: float
 
+
 app = FastAPI()
 analyzer = SentimentIntensityAnalyzer()
 
-@app.get("/", response_model=PolarityScores)
-def sentiment(phrase: str):
-    return analyzer.polarity_scores(phrase)
 
-@app.post("/", response_model=PolarityScores)
+@app.get("/", response_model=PolarityScores)
```

Comment:
curious what your thoughts are on supporting both `GET` and `POST`?

GET is def easier for testing things out but not nearly as secure as POST since the data is embedded in the url itself.

For FTS services, we autogenerate both GET and POST endpoints, but for python we need to decide between them and I think this will be a good question to have consensus on going forwards.

Reply:
I think we should support both, the reasoning for the removal here was based on the same function defined twice. Overall I still think it is up to the individual developer whether he/she needs only `GET` or `POST` is needed. 

Overall i propose we keep to the consensus of what those two tries to solve
- `GET` => get data from a resource
- `POST` => sending data to the server

Reply:
In this case there is reasons for both, since we are 'just' wrapping a function underneath, which solves exactly the same, so there is an argument for having both `POST` and `GET`, just with a different function name, that also follows the pretty standard of `Python` in this case